src/components/Movies/Movies.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import Movie from '../Movie/Movie.js';
+import './Movies.css';
+
+const Movies = ({movies}) => {
```

Comment:
I like the use of destructuring going on in the files. I'll keep this in mind as I code so that we are consistent.